src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,22 @@
+package main.java.core.basesyntax;
+
+import core.basesyntax.Ball;
+import core.basesyntax.ColorSupplier;
+import java.util.Random;
+
+public class Lottery {
+
```

Comment:
[Please don't add redundant empty lines to your code.](https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#please-dont-add-redundant-empty-lines-to-your-code)